.appveyor.yml

```diff
@@ -11,20 +11,32 @@ platform: x64
 configuration: Release
 
 install:
-  - "SET PATH=C:\\Miniconda-x64;C:\\Miniconda-x64\\Scripts;%PATH%"
+  - "SET PATH=C:\\Miniconda-x64;C:\\Miniconda-x64\\Scripts;C:\\Python27-x64;%PATH%"
   # by default, all script lines are interpreted as batch
   - cinst -y swig
-  - "conda update -y conda"
-  - "conda install -y numpy" # numpy installs mkl2017 !
+  - conda update -y conda
+  - conda install -y numpy # numpy installs mkl2017 !
```

Comment:
BTW yes we can install mkl2017 with conda, but no way to install the headers that we need (e.g mkl.h)